packages/labs/test-projects/test-element-a/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "incremental": true,
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
```

Comment:
This is the default

Reply:
There was an error without this. We may need to do some path massaging in the analyzer.